olive/model/__init__.py

```diff
@@ -831,8 +833,9 @@ def __init__(self, model_path: OLIVE_RESOURCE_ANNOTATIONS, model_attributes: Opt
             model_attributes=model_attributes,
         )
         # check if the model files (xml, bin) are in the same directory
-        if self.resource_paths["model_path"].is_local_resource():
-            _ = self.model_config
+        model_path = create_resource_path(self.resource_paths["model_path"])
```

Comment:
nit: `self.get_resource("model_path")` might be better?

Reply:
will use self.model_path